airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresQueryUtils.java

```diff
@@ -79,6 +81,11 @@ SELECT pg_relation_filenode('%s')
 
   public static final String TOTAL_BYTES_RESULT_COL = "totalbytes";
 
+  public static final String CTID_TABLE_BLOCK_SIZE =
```

Comment:
Can we comment what this block size is? Is it the page size or some other Postgres internal